.github/actions/compile-commit/action.yml

```diff
@@ -7,9 +7,23 @@ inputs:
 runs:
   using: composite
   steps:
+    - name: Get the hash of local working tree
```

Comment:
To be precise it gets hash of the tree at a commit - doesn't include changes in working tree.